src/System Application/App/BLOB Storage/src/PersistentBlobImpl.Codeunit.al

```diff
@@ -10,7 +10,7 @@ codeunit 4108 "Persistent Blob Impl."
     Access = Internal;
     InherentEntitlements = X;
     InherentPermissions = X;
-    Permissions = TableData "Persistent Blob" = rimd;
+    Permissions = tabledata "Persistent Blob" = rimd;
```

Comment:
Makes no sense to me that tabledata would be all lowercase, but I guess that's a choice someone made :-) Should we create a bug for the compiler?